serverless.yml

```diff
@@ -185,6 +185,12 @@ functions:
       - httpApi:
           path: /currency/{identifier}/addresses/{address}/fee-transactions/received
           method: GET
+  metagraphs:
+    handler: src/handler.metagraphs
+    events:
+      - httpApi:
+          path: /currency
```

Comment:
just my opinion but `/currency` seems somewhat confusing to retrieve a list of metagraphs that may not include a token (like a data application without fees). Two points I can think of to support this approach would be that 1) we imagine all metagraphs to include a custom token or 2) this is keeping with the historical approach for how metagraphs are labeled in other portions of the overall protocol (like the ML0 is technically a CurrencyL0). 

Not a strong preference to change this, just sharing my thoughts.

Reply:
@scasplte2 What do you mean by "token"?

Reply:
ah, I see what you mean. I think it should be discussed wider on slack and changed across all repositories if needed.

Reply:
> just my opinion but `/currency` seems somewhat confusing to retrieve a list of metagraphs that may not include a token (like a data application without fees). Two points I can think of to support this approach would be that 1) we imagine all metagraphs to include a custom token or 2) this is keeping with the historical approach for how metagraphs are labeled in other portions of the overall protocol (like the ML0 is technically a CurrencyL0). 
> 
> 
> 
> Not a strong preference to change this, just sharing my thoughts.

Fair point. The /currency was just a part of the ticket but I think we should make /metagraphs or something